packages/cfd/src/features/Components/proof-of-identity.tsx

```diff
@@ -0,0 +1,49 @@
+// @ts-expect-error remove this line when ProofOfIdentityContainerForMt5 is converted to TS
+import ProofOfIdentityContainerForMt5 from '@deriv/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container-for-mt5.jsx';
+import React from 'react';
+import { useStore, observer } from '@deriv/stores';
+import type { TCoreStores } from '@deriv/stores/types';
+
+type TCFDValue = {
+    poi_state: string;
+};
+
+type TFormValues = {
+    poi_state?: string;
+};
+
+export type TProofOfIdentity = {
+    index: number;
+    onSubmit: (index: number, value: TCFDValue) => void;
+    value: TCFDValue;
+    addNotificationMessageByKey: TCoreStores['notifications']['addNotificationMessageByKey'];
+    height: string;
+    onSave: (index: number, values: TFormValues) => void;
+    removeNotificationByKey: TCoreStores['notifications']['removeNotificationByKey'];
+    removeNotificationMessage: TCoreStores['notifications']['removeNotificationMessage'];
+    jurisdiction_selected_shortcode: string;
+};
+
+const ProofOfIdentity = observer(({ index, onSave, onSubmit, ...props }: TProofOfIdentity) => {
+    const { client } = useStore();
+    const { account_settings, residence_list } = client;
```

Comment:
let's avoid connecting the component to the store for data layer usage and use hooks instead